challenges/2022-10-11-blast-from-the-past/setups/ruby/info.md

```diff
@@ -0,0 +1,11 @@
+# Running Tests
+
+Run the tests to check your solution by navigating to this directory and running `ruby solution.rb`.
+
+# Requirements
+
+- [Ruby 2.7+](https://www.ruby-lang.org/en/downloads/)
+- Rspec (Test Runner)
+  - `gem install rspec`
```

Comment:
I got a permissions error with `gem install rspec`, but `gem install rspec --user-install` worked flawlessly. Do you mind adding that flag?

Reply:
Added it as a second option. Thanks!

Reply:
Added it as a second option, thanks!